src/modules/timeseries.ts

```diff
@@ -32,6 +33,33 @@ export function fetchTimeseries(assetId: number) {
     dispatch({ type: SET_TIMESERIES, payload: { items: results.items } });
   };
 }
+let searchTimeseriesRID = 0;
+// Functions
+export function searchTimeseries(query: string, assetId?: number) {
+  return async (dispatch: Dispatch<SetTimeseriesAction>) => {
+    searchTimeseriesRID += 1;
+    const id = searchTimeseriesRID;
+    const results = await sdk.post(
+      `/api/playground/projects/${sdk.project}/timeseries/search`,
```

Comment:
Do we have a plan for updating this once the API changes? So it doesn't suddenly break!

Reply:
Yes, this will still live in staging (not production)! until this is live into production, it will still be here.